.jenkins/test.sh

```diff
@@ -63,18 +62,25 @@ if [[ "$BUILD_ENVIRONMENT" == *-cuda* ]]; then
   EXTRA_TESTS+=(caffe2/contrib/nccl)
 fi
 
+# Add the site-packages in the caffe2 install prefix to the PYTHONPATH
+SITE_DIR=$($PYTHON -c "from distutils import sysconfig; print(sysconfig.get_python_lib(prefix=''))")
+export PYTHONPATH="${PYTHONPATH}:/usr/local/caffe2/$SITE_DIR"
```

Comment:
I think with your changes there is no need to export PYTHONPATH

Reply:
I thought so at first, but the default python points to `/usr/local/lib/pythonX.Y/site-packages`. So, if path to the caffe python module was `/usr/local/lib/pythonX.Y/site-packages/caffe2`, then it would work. 

However, looking at the script, it seems like your installing to to `/usr/local/caffe2` instead of `/usr/local`. This means the python module will exist at `/usr/local/caffe2/lib/pythonX.Y/site-packages/caffe2`, so it wont be in the `PYTHONPATH` by default (which I think is the correct behavior when install to `/usr/local` with a project-name-prefix).

To make this clear maybe its best to replace `/usr/local/caffe2` with a variable named `CAFFE2_INSTALL_PREFIX`? Then it would be easy to change where you install if you didn't want to install to the jenkins system to run tests.

Reply:
@Erotemic The default cmake install prefix should be /usr/local, thus with your changes, the caffe2 files will be installed into /usr/local/lib/pythonX.Y/site-packages/caffe2. I believe you can remove the PYTHONPATH export line.

Reply:
If you look in build.sh the CMAKE_INSTALL_PREFIX is set to "/usr/local/caffe2", so this line would be necessary. However, there still is the issue of two failing tests.